### ISSUE TITLE
Typo in Product Description?

### ISSUE THREAD

**Issue by richardfencel** [Microsoft/IPC#26]
Should "...is available which allows interaction with C++ as well." be instead "is available which allows interaction with C# as well."?

**Comment by ara-ayvazyan** [comment]
Not really. It means that the native and managed versions can communicate with each other.
